bridge-proxy/Cargo.toml

```diff
@@ -23,18 +23,21 @@ path = "../common/tx-batch-module"
 [dependencies.esdt-safe]
 path = "../esdt-safe"
 
+[dependencies.bridged-tokens-wrapper]
+path = "../bridged-tokens-wrapper"
+
 [dependencies.multiversx-sc]
-version = "=0.51.1"
+version = "=0.52.2"
 
 [dependencies.multiversx-sc-modules]
-version = "0.51.1"
+version = "0.52.2"
 
-[dependencies.adder]
+[dependencies.crowdfunding-esdt]
```

Comment:
why do we have this dependency ?

Reply:
For unit tests purposes (call a function from crowdfunding)